test/lib/bench_gateway/bench_gateway.go

```diff
@@ -189,13 +189,16 @@ func (gateway *BenchGateway) Backends() map[string]*testBackend.TestBackend {
 
 // MakeRequest helper
 func (gateway *BenchGateway) MakeRequest(
-	method string, url string, body io.Reader,
+	method string, url string, headers map[string]string, body io.Reader,
```

Comment:
Want to make all the header interfaces for http/tchannel the same where possible so going to use `map[string]string` in all 6 locations.